### ISSUE TITLE
Change key icon in a11y view

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#229827]
I had to re-read these to figure out the difference, so maybe we should consider using different icons for each action. But I think we should be using keyboard here instead of a key. Keys say to me password, authentication, etc. Not a key/hoykeys on a keyboard.

![Image](https://github.com/user-attachments/assets/1cff6784-cd82-46f6-b8cc-caf2379aca3e)
![Image](https://github.com/user-attachments/assets/5989b071-cfd9-4f36-a452-e00b26de2656)

**Comment by Tyriar** [comment]
Example icons I think would be better:

![Image](https://github.com/user-attachments/assets/3cb04e1f-3b6a-45cf-b358-aeeed0197455)

New icon idea for unassigned:

![Image](https://github.com/user-attachments/assets/c3cbae3a-d4ec-4dd1-9109-237bc93a33a0)

**Comment by meganrogge** [comment]
Blocked by https://github.com/microsoft/vscode-codicons/issues/263

**Comment by hbons** [comment]
@Tyriar are those existing icons? I don't think we have tin the set but I can create. Thinking about fewer keys though.

**Comment by meganrogge** [comment]
ideally, we'd have distinct keyboard icons. It might be awhile before we can have those, so considering this fixed with the switch from 🔑 to ⌨
